shared/constants/people.js

```diff
@@ -159,16 +158,17 @@ export const reduceRPCItemToPeopleItem = (
         })
       )
     }
-  } else if (item.data.t === 3) {
-    if (__DEV__ && flags.peopleAnnouncementsEnabled) {
-      const a: any = item.data.announcement
+  } else if (item.data.t === RPCTypes.homeHomeScreenItemType.announcement) {
+    const a = item.data.announcement
+    if (a) {
       return list.push(
         makeAnnouncement({
```

Comment:
looks like `makeAnnouncement` below is dropping some fields

Reply:
fixed